owncloudApp/src/main/java/com/owncloud/android/operations/SynchronizeFolderOperation.java

```diff
@@ -60,6 +63,10 @@
  * Does NOT enter in the child folders to synchronize their contents also, BUT requests for a new operation instance
  * doing so.
  */
+@Deprecated
+// Call RefreshFolderFromServerAsyncUseCase instead. Keep it for the moment.
```

Comment:
Include comment inside `@Deprecated` ?

This way we can see more info when moving the mouse over the deprecated object, without having to enter this class.

Reply:
Adding reason, replaceWith etc, are options available only in kotlin. So i cannot use them in java world 😭

Reply:
Yep, sorry